### PR TITLE
readme and changelog updates

### PR DESCRIPTION
**Pull request by LisaDelaney**

### CODE REVIEW

File: README.md
Comment (daineAMD) [inline README.md]: rocSPARSE is a dependency of rocSOLVER as well, not sure if that should be included here.

File: README.md
Comment (daineAMD) [inline README.md]: This sentence (prior to and after changes) is confusing to me, I'm not really sure what it's even trying to say. Something like the following may be better:
"hipBLAS GEMM can process matrices in batches with regular strides by using the strided-batched version of the API as follows:"

**Review by daineAMD — COMMENTED** [review]
Looks good, thanks for the update! May want to wait until after #714 is merged, will update after tomorrow morning's meeting.

File: README.md
Comment (LisaDelaney) [reply README.md]: Sure, I'll add it in

File: README.md
Comment (LisaDelaney) [reply README.md]: My interpretation was that the example included all possible options. But your edit allows room for future additions without content modification, so seems like a better option regardless.

**Comment by daineAMD** [comment]
@LisaDelaney 
I've merged release/rocm-rel-6.0 branch into develop so there are some merge conflicts here now. If you need any help resolving them let me know and I can update your PR.

File: CHANGELOG.md
Comment (daineAMD) [inline CHANGELOG.md:6]: This can just be `## hipBLAS 2.0.0 for ROCm 6.0.0` as it was updated with #714, sorry for the merge conflicts.
Otherwise looks good! After this change you can go ahead and merge, or I can if there's permission issues.

**Review by daineAMD — APPROVED** [review]

**Comment by daineAMD** [comment]
@LisaDelaney  I will go ahead with the merge of this PR. I'll be opening another to update the Changelog for next release soon. Thanks again!

**Pull request merged**
